pkg/reconciler/source/duck/resources/eventtype_test.go

```diff
@@ -49,31 +50,31 @@ func TestMakeEventType(t *testing.T) {
 			},
 		},
 		CeType:      "my-type",
-		CeSource:    "my-source",
-		CeSchema:    "my-schema",
+		CeSource:    apis.HTTP("my-source"),
+		CeSchema:    apis.HTTP("my-schema"),
 		Description: "my-description",
 	}
 
 	got := MakeEventType(args)
 
-	want := &v1alpha1.EventType{
+	want := &v1beta1.EventType{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      fmt.Sprintf("%x", md5.Sum([]byte("my-type"+"my-source"+"my-schema"+"source-uid"))),
+			Name:      fmt.Sprintf("%x", md5.Sum([]byte("my-type"+"http://my-source"+"http://my-schema"+"source-uid"))),
```

Comment:
looks harmless, but this is a funny search and replace.

Reply:
Yes, this is because we changed to URL.